packages/office-ui-fabric-react/src/components/Nav/Nav.base.tsx

```diff
@@ -185,10 +185,12 @@ export class NavBase extends BaseComponent<INavProps, INavState> implements INav
     const isLinkSelected: boolean = this._isLinkSelected(link);
     const isRtl: boolean = getRTL();
     const absolutePositionString = `${_indentationSize * nestingLevel + 1}px`;
+    const divProps: React.HTMLProps<HTMLDivElement> = { ...getNativeProps(link, divProperties) };
```

Comment:
Oh! actually, getNativeProps allows an exclusion list as well as the 3rd parameter.